rechunker/api.py

```diff
@@ -7,6 +7,7 @@
 import dask
 import dask.array
 import zarr
+from packaging.version import Version
```

Comment:
We should add `packaging` to our dependencies in the pyproject.toml instead of getting it transitively.